### PR TITLE
Fix osm_controller_args and osm_api_server_args settings.

### LINKED ISSUE #1106: Added option for resource-quota-sync-period
Hi,

if you want to use quotas, you have to set resource-quota-sync-period on kubernetesMasterConfig/controllerArguments (Documentation: https://docs.openshift.com/enterprise/3.1/dev_guide/quota.html#configuring_quota_sync_period).

It should be configurable via ansible.

### PR DESCRIPTION
**Pull request by abutcher**
Fixes #1106 

@detiber PTAL I wasn't able to set these prior to changes

### CODE REVIEW

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

File: roles/openshift_master/templates/master.yaml.v1.j2 [hunk 1]
```diff
@@ -87,8 +87,8 @@ kubernetesMasterConfig:
   - v1beta3
   - v1
 {% endif %}
-  apiServerArguments: {{ api_server_args if api_server_args is defined else 'null' }}
-  controllerArguments: {{ controller_args if controller_args is defined else 'null' }}
+  apiServerArguments: {{ openshift.master.api_server_args | to_json if openshift.master.api_server_args | default(None) else 'null' }}
+  controllerArguments: {{ openshift.master.controller_args | to_json if openshift.master.controller_args | default(None) else 'null' }}
```
Comment (detiber) [inline roles/openshift_master/templates/master.yaml.v1.j2]: you shouldn't need the if/else with using the default filter.

Comment (detiber) [inline roles/openshift_master/templates/master.yaml.v1.j2]: Does openshift.master.api_server_args | default(None) | to_json not work?

Comment (abutcher) [inline roles/openshift_master/templates/master.yaml.v1.j2]: Works great

**Comment by abutcher** [comment]
aos-ci-test

**Comment by detiber** [comment]
:+1: from me then, you can see how much testing went into this feature previously :)

**Comment by abutcher** [comment]
@brenton should be good to merge

**Pull request merged**